opencdc/metadata.go

```diff
@@ -313,6 +332,112 @@ func (m Metadata) SetPayloadSchemaVersion(version int) {
 	m[MetadataPayloadSchemaVersion] = strconv.Itoa(version)
 }
 
+// GetFileName gets the metadata value for key MetadataFileName.
+// If the value does not exist or is empty the function returns ErrMetadataFieldNotFound.
+func (m Metadata) GetFileName() (string, error) {
+	return m.getValue(MetadataFileName)
+}
+
+// SetFileName sets the metadata value for key MetadataFileName.
+func (m Metadata) SetFileName(filename string) {
+	m[MetadataFileName] = filename
+}
+
+// GetFileSize gets the metadata value for key MetadataFileSize.
+// If the value does not exist or is empty the function returns ErrMetadataFieldNotFound.
+func (m Metadata) GetFileSize() (int64, error) {
+	size, err := m.getValue(MetadataFileSize)
+	if err != nil {
+		return 0, err
+	}
+
+	filesize, err := strconv.ParseInt(size, 10, 64)
+	if err != nil {
+		return 0, fmt.Errorf("invalid filesize %q: %w", size, err)
+	}
+
+	return filesize, nil
+}
+
+// SetFileSize sets the metadata value for key MetadataFileSize.
+func (m Metadata) SetFileSize(filesize int64) {
+	m[MetadataFileSize] = strconv.FormatInt(filesize, 10)
+}
+
+// GetFileHash gets the metadata value for key MetadataFileHash.
+// If the value does not exist or is empty the function returns ErrMetadataFieldNotFound.
+func (m Metadata) GetFileHash() (string, error) {
+	return m.getValue(MetadataFileHash)
+}
+
+// SetFileHash sets the metadata value for key MetadataFileHash.
+func (m Metadata) SetFileHash(filehash string) {
+	m[MetadataFileHash] = filehash
+}
+
+// GetFileIsChunked gets the metadata value for key MetadataFileHash.
+func (m Metadata) GetFileChunked() bool {
+	chunked, err := m.getValue(MetadataFileChunked)
+	if err != nil {
+		return false
+	}
+	if chunked == "true" {
+		return true
+	}
+	return false
```

Comment:
I'd like to suggest to follow a similar approach as other methods, where we return an error, if we can't get the value. We can also use ``strconv.ParseBool` (it handles all the different letter case, and also handles 0, 1, t, f, etc.)

Reply:
updated.